docs/aeternity-erlang-compatibility.md

```diff
@@ -0,0 +1,201 @@
+# aeternity Erlang implementation compatibility
```

Comment:
Shouldn't we make a test framework for this?
We could download epoch via the makefile and create a wrapper for ExUnit to directly invoke and test code running in epoch.

Reply:
Yes, there is already and issue for this https://github.com/aeternity/elixir-node/issues/574